src/constants.js

```diff
@@ -0,0 +1,6 @@
+export const accessibilityTransformations = {
+  'darkmode': {effect: 'tint:75:black'},
+  'brightmode': {effect: 'tint:50:white'},
```

Comment:
how does this solve the bright/dark mode case in general? Dark mode and bright mode is more complex than just using a constant `tint` though

Reply:
Agree @mayashavin , but we're starting with simple implementation and we will enhance based on your feedback